### PR TITLE
VACMS-9129 abstract Paragraph component

### PR DESCRIPTION
**Pull request by penny-lischer**
- created a Paragraph component to delegate which component to render based on the Paragraph data.
- Fixed lint errors on the news_story tests

### CODE REVIEW

File: src/components/paragraph/index.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+export function Paragraph({ paragraph, ...props }: ParagraphProps) {
+  if (!paragraph) return null
+
+  const Component = paragraphMeta[paragraph.type].component
+
+  if (!Component) return null
+
+  return <Component paragraph={paragraph} {...props} />
+}
```
Comment (amponce) [inline src/components/paragraph/index.tsx:82]: This looks great! is it working as expected?

File: src/types/paragraph.d.ts [hunk 2]
```diff
@@ -10,9 +10,20 @@ import {
...
+  NodeOffice,
+  NodePersonProfile,
+  NodePromoBanner,
   NodeQA,
+  NodeRegionalHealthCareServiceDes,
+  NodeStepByStep,
+  NodeStoryListing,
+  NodeSupportResourcesDetailPage,
   NodeSupportService,
 } from './node'
```
Comment (amponce) [inline src/types/paragraph.d.ts:28]: Why do we have `node` and `media` referenced in paragraph.d.ts? I haven't looked at this very closely but it seems a bit strange

File: src/types/paragraph.d.ts [hunk 3]
```diff
@@ -143,6 +181,19 @@ export interface ParagraphServiceLocationAddress extends DrupalParagraph {
...
+  field_first_name: fieldFirstName
+  field_last_name: fieldLastName
+  field_email_address: fieldEmailAddress
+  field_phone_number: fieldPhoneNumber
+  field_suffix: fieldSuffix
+  field_description: fieldDescription
+  field_complete_biography_create: fieldCompleteBiographyCreate
+  field_media: fieldMedia
+  field_entity: fieldEntity
+}
```
Comment (amponce) [inline src/types/paragraph.d.ts:195]: Doesn't need to be this ticket but at some point we'll need descriptions so type doc picks everything up. Maybe we should create a task?

File: src/types/paragraph.d.ts (hunk 2 above)
Comment (penny-lischer) [reply src/types/paragraph.d.ts:28]: Will fix. This happened when merging with main.

**Pull request merged**